docs/pages/accounts/sso.mdx

```diff
@@ -21,6 +26,34 @@ Expo SSO supports the following identity providers:
 
 We implement the [OpenID Connect Discovery 1.0](https://openid.net/specs/openid-connect-discovery-1_0.html) specification and are working to verify additional compatible identity providers. If you use another identity provider and are interested in SSO, [let us know](https://expo.dev/contact).
 
+## Enable SSO
+SSO is enabled by creating SSO configuration for the Organization.
+
+<Step label="1">
+	Log in as the Organization account owner, select your Organization Account, then go to Organization Settings > Overview.
+</Step>
+
+<Step label="2">
+	Click the ‘Start’ button next to the ‘Create SSO configuration for account’ option.
+</Step>
+
+<Step label="3">
+	Enter the configuration details for your IdP in the form, with the information from setting up your IdP:
+  - your client ID
+  - your client secret
+  - your IdP subdomain/tenant ID, if needed. Check the Issuer field help for what to enter.
+</Step>
+
+<Step label="4">
+	Click ‘Create SSO Configuration’ and you should see the message that you have successfully created SSO configuration for your account.
+	</Step>
+
+<Step label="5">
+  The Organization Settings > Overview page will now display an ‘Update SSO configuration’ option.
```

Comment:
```suggestion
  The **Organization settings** > **Overview** page will now display an **Update SSO configuration** option. Use this option to update the client secret when it changes.
```

In Expo dashboard, "Organization settings" is the name of the tab. For consistency, let's stick to using the same form.

Reply:
Done